requirements.txt

```diff
@@ -7,6 +7,9 @@ requests
 # Optional NLP tools
 nltk
 spacy
+# Needed for machine translation tokenizers
+https://github.com/explosion/spacy-models/releases/download/de_core_news_sm-2.2.5/de_core_news_sm-2.2.5.tar.gz#egg=de_core_news_sm==2.2.5
```

Comment:
What's this for? If spacy is not installed for a specific language, it will throw out a error message.

Reply:
Initially I added this for resolving issues in unittest as the dataset used was primarily dutch and the I was using spacy as the tokenizer. But after do some thinking maybe this is unnecessary to be included in requirements.txt